bot/exts/info/pypi.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from random import choice
+
+from discord import Embed
+from discord.ext.commands import Cog, Context, command
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+URL = "https://pypi.org/pypi/{package}/json"
+FIELDS = ["author", "requires_python", "summary", "license"]
+
+log = logging.getLogger(__name__)
+
+
+class PyPi(Cog):
+    """Cog for getting information about PyPi packages."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @command(name="pypi", aliases=("package", "pack"))
+    async def get_package_info(self, ctx: Context, package: str) -> None:
+        """Getting information about a specific package."""
+        embed = Embed(title=choice(NEGATIVE_REPLIES), colour=Colours.soft_red)
```

Comment:
Can we add the pypi thumbnail as: (Pydis Emoji)
Link: `https://cdn.discordapp.com/emojis/766274397257334814.png`
![Emoji from pydis](https://cdn.discordapp.com/emojis/766274397257334814.png)

Reply:
I can probably find a permanent url, I don't think a discord link would work out in the long term.

Reply:
I couldn't find a permanent url or one in the branding repo, so I'll just go with the Discord link for now.

Reply:
Well i could only find this
Link: `https://commons.wikimedia.org/wiki/File:PyPI_logo.svg`
![image](https://user-images.githubusercontent.com/69356296/107766558-a30d3200-6d59-11eb-8f92-423b4bbb448a.png)